libfuse/fs.go

```diff
@@ -199,18 +199,22 @@ func (f *FS) Root() (fs.Node, error) {
 
 // Statfs implements the fs.FSStatfser interface for FS.
 func (f *FS) Statfs(ctx context.Context, req *fuse.StatfsRequest, resp *fuse.StatfsResponse) error {
-	// TODO: Fill in real values for these.
-	var bsize uint32 = 32 * 1024
 	*resp = fuse.StatfsResponse{
-		Blocks:  ^uint64(0) / uint64(bsize),
-		Bfree:   ^uint64(0) / uint64(bsize),
-		Bavail:  ^uint64(0) / uint64(bsize),
-		Files:   0,
-		Ffree:   0,
-		Bsize:   bsize,
+		Bsize:   fuseBlockSize,
 		Namelen: ^uint32(0),
-		Frsize:  0,
+		Frsize:  fuseBlockSize,
 	}
+	status, _, err := f.config.KBFSOps().Status(ctx)
```

Comment:
Ah that makes sense. I guess we'd have to do time based cache then. Or perhaps on top of that, if pid points to a df binary, make sure to hit the bserver?